Lib/test/_test_multiprocessing.py

```diff
@@ -3878,6 +3878,10 @@ class OptionalAttachSharedMemory(shared_memory.SharedMemory):
 
         sms.close()
 
+        # Test creating a shared memory segment with size 0
```

Comment:
If there are no existing tests for negative numbers, they should be added. Also, this test should explicitly pass in zero and not rely on the default value. Or perhaps test it both ways: with the default and with an explicit zero.

Reply:
will do that.

Reply:
done.